Makefile

```diff
@@ -340,7 +340,7 @@ style :
             --suffix=none --preserve-date --formatted \
             --exclude=include/ceedf.h \
             include/*.h interface/*.[ch] tests/*.[ch] backends/*/*.[ch] \
-            examples/*/*.[ch] examples/*/*.[ch]pp
+            examples/*/*.[ch] examples/*/*.[ch]pp -i
```

Comment:
exclude was throwing an `(unmatched)` error and stopping, so I added `-i` to not stop 'make style' on an exclude error